torch/fx/graph.py

```diff
@@ -65,10 +65,22 @@ def map_arg(a: Argument, fn: Callable[[Node], Argument]) -> Argument:
     else:
         return a
 
+class WithInsertPoint:
```

Comment:
PyTorch convention is for resource guards to use snake case. `With` is already in the syntax for using it.  Probably should be more descriptive:
`with insert_before(n):`